cmd/controllerinit/run.go

```diff
@@ -89,3 +79,16 @@ func runControllerCmd(cmdName string, args []string) error {
 
 	return cmd.Run()
 }
+
+func configureSystem() error {
+	globalConfigurer, err := global.NewGlobalConfigurer()
+	if err != nil {
+		return fmt.Errorf("creating configurer: %v", err)
+	}
+
+	if err := globalConfigurer.Configure(); err != nil {
```

Comment:
lets stick to single style of 

err := ...
if err != nil {
  ...

instead of shortened style.